synfig-core/src/synfig/filesystemnative.h

```diff
@@ -95,6 +101,23 @@ namespace synfig
 		virtual FileSystem::ReadStream::Handle get_read_stream(const String &filename);
 		virtual FileSystem::WriteStream::Handle get_write_stream(const String &filename);
 		virtual String get_real_uri(const String &filename);
+#ifdef _WIN32
+		// Windows uses UTF-16 for filenames, so we need to convert it from UTF-8 before using it.
+		static std::wstring path(const std::string& path) {
```

Comment:
Why is the definition in the header and not in the .cpp?

Reply:
Agree.